karate-tests/src/test/java/mocks/stubs/errorResponses/RESOURCE_NOT_FOUND.json

```diff
@@ -1,19 +1,19 @@
 {
-  "resourceType": "OperationOutcome",
   "issue": [
     {
-      "severity": "error",
       "code": "not-found",
       "details": {
         "coding": [
           {
-            "system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
-            "version": "1",
             "code": "RESOURCE_NOT_FOUND",
-            "display": "Resource not found"
+            "display": "Resource not found",
+            "system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
+            "version": "1"
           }
         ]
-      }
+      },
+      "severity": "information"
```

Comment:
Is "information" accurate for this?

Reply:
that's what gets returned, it's not my code...